amps/ags/rm-automation/rm-automation-community-rest-api/src/test/java/org/alfresco/rest/rm/community/utils/AlfrescoRetryAnalyzer.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * #%L
+ * Alfresco Records Management Module
+ * %%
+ * Copyright (C) 2005 - 2024 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * -
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ * -
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * -
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ * -
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.rest.rm.community.utils;
+
+import org.testng.ITestResult;
+import org.testng.util.RetryAnalyzerCount;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class AlfrescoRetryAnalyzer extends RetryAnalyzerCount
+{
+    private int count = 4;
+
+    public boolean retry(ITestResult result)
```

Comment:
If we're overriding methods we should always make this clear via the **@Override** annotation. This is not only useful to better understand at first glance what is going on, but also provides compilation time feedback in case the parent signature changes in the future. This class will silently break and no one would ever know if someone changes the signature of the method we're overriding since we're not using the appropriate marker annotation.